modules/@apostrophecms/doc/index.js

```diff
@@ -937,6 +937,248 @@ module.exports = {
           return self.db.insertOne(self.apos.util.clonePermanent(doc));
         });
       },
+      // Set meta data for a given field, that will be live under `aposMeta`
+      // doc property. It returns the path to the meta property withouth the
+      // key. See `getMetaPath` method for more information.
+      //
+      // Signature:
+      // `apos.doc.setMeta(doc, namespace, [subobject], ...pathComponents, key, value);`
+      // where arguments are as follows:
+      // - `doc`: the document to attach the meta property to.
+      // - `namespace`: the namespace of the meta property, by convention the
+      //   module name that is setting the meta property.
+      // - `subobject`: (optional) the name of the field subobject (e.g. array
+      //   item, widget, or any other field type object that have `_id` property).
+      //   This argument dictates how `pathComponents` are interpreted. If
+      //   `subobject` is not provided, `pathComponents` are interpreted as
+      //   top-level meta properties. If `subobject` is provided, `pathComponents`
```

Comment:
"If `subobject` is not provided, `pathComponents` are interpreted as top-level meta properties"

How about "are interpreted as a path starting from `doc`" to make clear you can still have multiple components in this case